cmd/dashboard.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	selector         = "name=nginx-ingress-controller"
+	ingressNamespace = "kube-system"
+)
+
+var dashboardCmd = &cobra.Command{
+	Use:   "dashboard FLAG",
+	Short: "Opens the KubeApps Dashboard",
+	Long:  "Opens the KubeApps Dashboard",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		config, err := buildOutOfClusterConfig()
+		if err != nil {
+			return err
+		}
+
+		clientset, err := kubernetes.NewForConfig(config)
```

Comment:
There is nothing wrong with this clientset but if I were you I would try to use the c.ClientPool and c.ClientDiscovery from the `restClientPool()` function in order to reduce the dependency.

Reply:
Yeah that makes sense, I did see it but for some reason thought it was related to kubecfg so didn't use that. Will try to use it and update.